homeassistant/components/home_connect/quality_scale.yaml

```diff
@@ -0,0 +1,78 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling:
+    status: done
```

Comment:
I think we should change this to todo considering what we know about the limits and the current handling. It's too easy at the moment to run in to rate limits.

Reply:
```suggestion
    status: todo
```

Reply:
Do you think that now that #139379 has been merged, we can leave this as done?

Reply:
There's still the risk of getting blocked after 1000 r/day. I think we need to address that first.

Reply:
I think that with #142615 and #143509 the only cases that reach 1000 calls in less than a day are the following:

- The user uses uses this integration functionality too much (I think that it would be fair that the user gets API limited then)
- The user uses too many times the refresh entity service (which I don't see why the user would do that, I also think that it would be fair that the user gets API limited)
- The user constantly reloads the integration (fair also)
- More than one appliance is disconnecting and connecting back again less than 8 times per hour

Do you have in mind more any other possible cases?

Reply:
I think the main problem for 1000 r/day is the flaky device disconnecting/connecting too frequently. I haven't seen other examples problematic for that limit.

Reply:
So it's okay now to leave `appropriate-polling` as `done`?

Reply:
For now, I think so.